content/01.main_content.md

```diff
@@ -34,15 +33,14 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
 
 Recognizing the aforementioned shortcomings, we propose the following recommendations for the creation and deployment of automated auditing tools:
   1. *Transparency.* Publicly provide all tools, code, and data used in the analysis.
-  This is to enable public scrutiny and transparency to those being audited and those whose data you are using.
-  If the data is private, we recommend providing de-identified or aggregated data. 
+  This enables public scrutiny to those being audited and those whose data you are using.
+  For confidential data, we recommend providing de-identified or aggregated data. 
 
-  2. *Individuals know best.* Self-identified demographic information should be used in preference to automated predictions.
+  2. *Individuals know best.* Self-identified demographic information should be used in preference to algorithmic predictions.
 
   3. *Aggregates only.* Audit results should not affect individuals.
-  Gender expression and ethnicity of an individual are not for an algorithm to decide nor are they the focus of an audit. 
-  The analyses must focus only on aggregate estimations and any intermediary predictions on an individual should not be used independently.
-  In addition, analysts must be mindful of hidden subpopulations that may be obscured when calculating aggregate statistics.
+  Analyses must focus on aggregate estimations and any intermediary individual predictions should not be used externally.
+  In addition, analysts must be mindful of hidden subpopulations that may be obscured during aggregation.
+
+  4. *Inform the public.* Internal audits can help institutions reflect, however it does not give the public an opportunity to hold them accountable.
```

Comment:
This sentence seems a little strange at the moment. What do you think of "While internal audits can help institutions reflect, the public must be given an opportunity to hold them accountable." ? It's one fewer word

Reply:
yes sounds better: https://github.com/greenelab/disparities_opinion/pull/8/commits/30b8a87587fddf7e522503ce07829c02240e7602